string tellme         # collision or not
